modules/ROOT/pages/otel-support.adoc

```diff
@@ -0,0 +1,142 @@
+Following this, the trace displays the other two batch steps, along with their respective components.
+Finally, the Batch On Complete span is shown, along with the Logger component that appears after the Batch Job in the flow. 
+
+=== Example of Distributed Tracing Feature
+
+Consider two Mule apps where the first sends a request to the second.
+
+image::otel-mule-app1.png["A flow including a Listener, Logger, and Request components of an HTTP app"]
+
+image::otel-mule-app2.png["A flow showing the process flow for an asynchronous application"]
+
+Even though these are two different flows, distributed tracing captures both as part of the same trace because OpenTelemetry is instrumentation in Mule runtime and HTTP Connector.
+
+=== Errors in Tracing
+
+If the execution of the unit of work represented by a span produces an error, the span's status is set to `Error`. MuleSoft follows the semantic convention defined by OpenTelemetry that adds one or more `exception` events to the span:
+
+* `exception.type`: Mule error type
+* `exception.message`: Mule error message
+* `exception.stacktrace`: Mule flow stack string representation
+
+In the following example, when an error is produced, the span corresponding to the Raise Error component shows an error status in the Lightstep platform. 
+
+image::otel-errors-in-tracing.png["A flow of a telemetry flow with error handling"]
+
+This status is propagated to the parent span, for example, the flow span. A detailed view of the span shows the error status and the exception events.
+
+=== HTTP Conventions
+
+The HTTP Listener and the HTTP Request operations contribute additional attributes to the spans, providing more context for tracing the HTTP interactions.
+
+For flows with an HTTP Listener:
+
+* Span name: Combination of HTTP Method and path. For example, `POST /api`.
+* Span kind: `SERVER`.
+
+Additional attributes for an HTTP Listener associated flow span:
+
+[%header%autowidth.spread]
+|===
+| Attribute | Description 
+| `net.host.name` | The host address of the HTTP Listener
+| `net.host.port` | The port of the HTTP Listener
+| `http.user_agent` | The user agent of the received request
+| `http.route` | The path of the listener
+| `http.method` | the HTTP Method
+| `http.flavor` | the HTTP version
+|===
+
+For flows with an HTTP Request: 
+
+* Span name: HTTP Method. For example, `GET`
+* Span kind: `CLIENT`
+
+Additional attributes for an HTTP Request associated flow span:
+
+[%header%autowidth.spread]
+|===
+| Attribute | Description 
+| `http.url` | The target URL
+| `net.host.port` | The port of the HTTP Request
+| `net.peer.name` | The target IP address
+| `http.route` | The path of the HTTP Request
+| `http.flavor` | the HTTP Version
+|===
+
+== See Also
+
+* xref:monitoring::telemetry-exporter.adoc[Export Telemetry Data to Third-Party Monitoring Systems]
```

Comment:
This should link to the main section of the Anypoint Monitoring documentation, not just how to configure the exporter. Our job here is to explain how the flow and the distributed tracing works taking into account Mule specifics.